pkg/controller/daemon/daemon_controller.go

```diff
@@ -646,6 +646,12 @@ func (dsc *DaemonSetsController) deletePod(obj interface{}) {
 	glog.V(4).Infof("Pod %s deleted.", pod.Name)
 	dsc.expectations.DeletionObserved(dsKey)
 	dsc.enqueueDaemonSet(ds)
+
+	// delete or update ds
+	if ds.Status.NumberMisscheduled > 0 && len(pod.Spec.NodeName) != 0 {
+		// If scheduled pods were deleted, requeue suspended daemon pods.
+		dsc.requeueSuspendedDaemonPods(pod.Spec.NodeName)
+	}
```

Comment:
This can be combined with L635-L641

Reply:
I am not understand your idea very well, i think here should be the correct location, can you give me some more guidance?